stages/org.osbuild.firewall

```diff
@@ -92,6 +117,14 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_sources in sources:
+        zone = zone_sources.get("zone", "")
+        subprocess.run(["chroot",
+                        tree,
+                        "firewall-offline-cmd", f"--zone={zone}"] +
```

Comment:
Does `--zone=` work like not specifying the zone? This is a bit confusing. Can't we omit the argument if no zone was specified?

Reply:
it applies to the default zone if you do "--zone="""